packages/shopping/src/__tests__/acceptance/authentication.acceptance.ts

```diff
@@ -196,7 +200,8 @@ describe('authentication services', () => {
   });
 
   async function setupApp() {
-    app = await setupApplication();
+    const appWithClient = await setupApplication();
```

Comment:
@raymondfeng , how were these tests working before if the wrong object was being assigned to `app`?

Reply:
Haha, the test was moved from `unit` to `acceptance` and it used to have a different `helper`.